### ISSUE TITLE
Delete vertex V with LIMIT problems

### ISSUE THREAD

**Issue by andreyvk** [orientechnologies/orientdb#4148]
Hi,

I am using OrientDB 2.0.8. It seems that there's a problem deleting vertices when ```LIMIT``` is used in the command. This particular command ```delete vertex V limit 10``` throws the following error in the log:

```
2015-05-13 18:47:17:601 SEVERE Internal server error:
com.orientechnologies.orient.core.sql.OCommandSQLParsingException: Error on parsing command at position #0: Class 'LIMIT was not found [ONetworkProtocolHttpDb]
```

Is there anything wrong that I am doing here?

**Comment by andreyvk** [comment]
A temporary workaround could be done if some class extends ```V``` in which case the following command works fine:

```
delete vertex V where @class="Foo" limit 10
```

Apparently where's some issue with the parser.

**Comment by nagarajasr** [comment]
Can you try with OrientDB 2.1? The parser has been rewritten in that release.

**Comment by andreyvk** [comment]
Downloaded and tried 2.1-rc2. It's the same

**Comment by luigidellaquila** [comment]
fixed in both 2.0.x (will be released with 2.0.10) and develop (2.1 RC3)

**Comment by andreyvk** [comment]
Great thanks!
